packages/react-percy-ci/src/index.js

```diff
@@ -31,6 +33,24 @@ export default async function run(percyConfig, webpackConfig, percyToken) {
     return;
   }
 
+  if (percyConfig.debug) {
+    const html = getHTML(assets);
+    const htmlPath = path.join(percyConfig.rootDir, '.percy-debug', 'index.html');
+    fs.writeFileSync(htmlPath, html);
+    reporter.log(
+      'Debug snapshots by opening %s in your browser and using the following query strings:',
+      chalk.blue(htmlPath),
```

Comment:
Maybe prepend `file://` here so people can open it directly?

Reply:
Only reason I didn't was for Windows users since it would need to be backslashes instead. I could detect the OS and format it appropriately though!

Reply:
I think my memory was wrong here, it was just older IE versions that didn't handle `file://` properly. Went ahead and added it to the console output.

![image](https://user-images.githubusercontent.com/733696/30326637-2b836fa6-9797-11e7-989f-d67598615205.png)